docs/source/app-dev/app-arch.rst

```diff
@@ -170,3 +170,10 @@ How is this used in practice?
 - Set ``min_ledger_time_abs`` or ``min_ledger_time_rel`` if the duration of command interpretation and transmission is likely to take a long time relative to the tolerance interval set by the ledger.
 - In some corner cases, the participant node may be unable to determine a suitable ledger time by itself. If you get an error that no ledger time could be found, check whether you have contention on any contract referenced by your command or whether the referenced contracts are sensitive to small changes of ``getTime``.
 
+More precisely, the ledger time is implemented as follows:
```

Comment:
I would suggest a more radical intervention into the body of this chapter. I would split up the section below and fold its pieces close to where respective subjects are already defined. For instance
lt_TX could be introduced in line 158
rt_TX could be introduced in line 162 where we talk about ledger's "system time" and the "real time". We have 3 terms to describe the same thing and we never really introduce any of them properly. This can confuse people.
Same could be done to skews which are related to "tolerance interval" but we never say it, leaving the reader guessing.

Reply:
Good point, there is a lot of overlap. The new text basically only describes in more detail what was already written in vague terms.

The existing section was intentionally only very high level, and intended for new application developers. I could add the technical details into the existing body, but IMO then the text would too technical for this page.

Maybe the technical details should be described in even more details and go into a separate "background concepts" page? At the same time, we could add the terms such as "transaction", "record time", "ledger (effective?) time", and "participant" to the [glossary](https://docs.daml.com/concepts/glossary.html).